operators/test/e2e/apm/standalone_test.go

```diff
@@ -28,3 +29,26 @@ func TestApmStandalone(t *testing.T) {
 	test.Sequence(nil, test.EmptySteps, apmBuilder).
 		RunSequential(t)
 }
+
+func TestApmStandaloneNoTLS(t *testing.T) {
+	apmBuilder := apmserver.NewBuilder("standalone-no-tls").
+		WithNamespace(test.Namespace).
+		WithVersion(test.ElasticStackVersion).
```

Comment:
`test.Namespace` and `test.ElasticStackVersion` are already set by default. I'm OK to set them explicitely but I am a little embarrassed that we are not consistent through all the tests.

Reply:
Idem for `test.DefaultSecurityContext()`.

Reply:
and so is `PodSecurityContext`